Tests/ComposableArchitectureTests/EffectIdentifierTests.swift

```diff
@@ -27,7 +27,7 @@
       XCTAssertNotEqual(id1, id2_2)
       XCTAssertNotEqual(id2_1, id2_2)
     }
-
+    #if compiler(>=5.5)
```

Comment:
Local property wrappers are available in Swift 5.4, contrary to the commit's message, but Xcode 12.5.1 had an issue with them (see https://forums.swift.org/t/segmentation-fault-11-with-local-property-wrappers/47905).
I bumped the requirement to Swift 5.5 to avoid the problem.